fenix/app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -698,6 +717,16 @@ class SettingsFragment : PreferenceFragmentCompat() {
         }
     }
 
+    @VisibleForTesting
+    internal fun setupInstallAddonFromFilePreference(settings: Settings) {
+        with(requirePreference<Preference>(R.string.pref_key_install_local_addon)) {
+            isVisible = settings.showSecretDebugMenuThisSession &&
+                isNightlyOrDebugOrBeta()
+        }
+    }
+
+    internal fun isNightlyOrDebugOrBeta() = Config.channel.isNightlyOrDebug || Config.channel.isBeta
```

Comment:
Nit: not sure if introducing this variable is worthwhile

Reply:
Removed